src/services/event.ts

```diff
@@ -98,8 +98,17 @@ async function findEventHelper(where, plural = false): Promise<EventShape | Even
     include: [
       {
         model: SessionReportPilot,
+        attributes: [
+          'id',
+          'eventId',
+          'data',
+          'createdAt',
+          'updatedAt',
+          [sequelize.literal('Date(CASE WHEN "SessionReportPilot".data->>\'startDate\' = \'\' THEN NULL ELSE "SessionReportPilot".data->>\'startDate\' END)'), 'startDate'],
```

Comment:
Please use backticks and add newlines so it is more readable and then the single quotes don't need to be escaped

Reply:
Also, don't use CASE for this, use NULLIF(<value>, ''), as this is far easier to read and is much faster